devopstools-jenkins-chefhabitat-kubernetes/azuredeploy.json

```diff
@@ -11,20 +11,13 @@
         "azureUsername": {
             "type": "string",
             "metadata": {
-                "description": "azure login name"
+                "description": "azure portal login username"
```

Comment:
Why does the user have to supply their Azure credentials to the deployment?  Seems like this should be a service principal

Reply:
Azure login credentials are required to create Kubernetes cluster using the Azure CLI.
Please refer below link:

https://docs.microsoft.com/en-us/azure/container-service/kubernetes/container-service-kubernetes-service-principal#option-2-generate-a-service-principal-when-creating-the-cluster-with-az-acs-create